examples/test/PivotTableMenu_test.js

```diff
@@ -66,6 +65,7 @@ fixture("Pivot Table Menu")
     .beforeEach(async t => {
         await loginUsingLoginForm(`${config.url}/hidden/pivot-table-dynamic`)(t);
 
+        await t.wait(1000);
```

Comment:
Would `await waitForPivotTableStopLoading(t);` work as well?

Reply:
Nice tip, I'll give it a try on Monday, I'll check if table load finished is tied with controls refresh.